pkg/daemon/ceph/client/image.go

```diff
@@ -63,6 +64,24 @@ func ListImages(context *clusterd.Context, clusterName, poolName string) ([]Ceph
 	return images, nil
 }
 
+func getImageInfo(context *clusterd.Context, clusterName, name, poolName string) (*CephBlockImage, error) {
+	imageSpec := getImageSpec(name, poolName)
+	args := []string{"info", imageSpec}
+	buf, err := ExecuteRBDCommand(context, clusterName, args)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get image %s info: %+v", imageSpec, err)
+	}
+
+	var image CephBlockImage
+	if err = json.Unmarshal(buf, &image); err != nil {
+		return nil, fmt.Errorf("unmarshal failed: %+v. raw buffer response: %s", err, string(buf))
+	}
+
+	image.Name = image.InfoName
```

Comment:
The image name is read from the json field `name` into `image.InfoName` and then copied to `info.Name`, as expected by existing code.

Reply:
do we need to conditionally copy in case the `name` is set, but the info name is not set? such as:
```
if image.Name == "" {
  image.Name = image.InfoName
}
```

Reply:
I did that at first, but then realized that `rbd info` always returns the image name in the `name` field, not `image`, so `image.Name == ""` is always true.